build/pbm-entry.sh

```diff
@@ -14,6 +14,12 @@ fi
 
 export PBM_MONGODB_URI
 
+mongo_shell=mongosh
```

Comment:
After the following changes, we can remove the following lines 17-66 at all @egegunes @inelpandzic what do you think?